docs/resources/cluster.md

```diff
@@ -551,6 +595,8 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the Cluster (string)
 * `agent_env_vars` - (Optional) Optional Agent Env Vars for Rancher agent. Just for Rancher v2.5.6 and above (list)
+* `cluster_agent_deployment_customization` - (Optional) Optional customization for cluster agent. Just for Rancher v2.7.5 and above (list)
+* `fleet_agent_deployment_customization` - (Optional) Optional customization for fleet agent. Just for Rancher v2.7.5 and above (list)
```

Comment:
The above example says cluster agent customization is for rancher v2.7.x and above, but it says v2.7.5 and above here.

Reply:
@jiaqiluo Nit question, but I noticed there's a lot of old docs where we say `Rancher v2.x.x or above` vs `and above` but the most recent docs use `and above` except for the Terraform docs examples. Is this something I'd want to upgrade to have all the terminology in sync? Seems like we could just use `and` moving forward and that'd be fine but I almost want to change the code example descriptions haha

Reply:
Sorry to be late to see your reply, but it looks like you have figured it out. Thank you for fixing it!